### DISCUSSION TITLE
Load models from local directory for SamGeo2

### DISCUSSION THREAD

**Discussion by amruthaprasad** [opengeos/segment-geospatial#368] [Q&A] [answered]
For SamGeo2, how do I bypass downloading models from huggingface and provide a link to the model that exists locally?

**Comment by giswqs** [comment, marked as answer]
Put the model under the `~/.cache/torch/hub/checkpoints` directory.

**Comment by amruthaprasad** [comment]
Thank you. But, that does not seem to work for me. I've installed segment-geospatial in a conda environment. Would the cache be at a different location?

**Reply by giswqs** [reply]
Run the code, it will show you where it is downloading to. Once you know the location, you can restart the kernel to stop the download.
